tests/test_infra.py

```diff
@@ -67,6 +67,7 @@ def test_setup_version():
     assert output.strip() == __version__
 
 
+@pytest.mark.xfail(reason="Should fail while migrating commands")
```

Comment:
consider using `strict=True` so we don't forget to revisit this when migration is done

Reply:
Ope, yeah that was an oversight on my part. I've [requested a lint rule](https://github.com/m-burst/flake8-pytest-style/issues/267) for it since it makes sense for us to enforce a value here.